### DISCUSSION TITLE
Can NumericUpDown represent a hexadecimal value?

### DISCUSSION THREAD

**Discussion by hanmarco** [MahApps/MahApps.Metro#4202] [Q&A]
## Describe the feature
`NumericUpDown` has StringFormat property, so i wrote `StringFormat="{}{0:X}"`
But formatException occured.

`NumericUpDown` doesn't support hexadecimal display?

**Comment by punker76** [comment]
@hanmarco This should be possible with latest version. Here is a short test

```xaml
<Grid xmlns:mah="http://metro.mahapps.com/winfx/xaml/controls">
    <mah:NumericUpDown x:Name="NUD"
                       Margin="10"
                       Maximum="255"
                       Minimum="0"
                       StringFormat="{}{0:X}"
                       Value="255" />
</Grid>
```

![image](https://user-images.githubusercontent.com/658431/138851614-4ed7a90d-f9f5-4241-a194-0281c0ac9b51.png)
